be/src/exec/pipeline/connector_scan_operator.cpp

```diff
@@ -0,0 +1,181 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
```

Comment:
if this file is moving from hdfs_scan_operator.cpp, it's better to use `git mv` to track the movement of files.

Reply:
It's just move:
1. `hdfs_scan_operator` contains two parts: a. scan operator related code b. hdfs data souce related code
2. `connector_scan_operator` focus on scan operator related code
3. `hive data source` focus on hdfs data source reloated code

And if you look close, you can see `connector_scan_operator` is very generic comparing to `hdfs_scan_operator`